apps/fabric-website-resources/src/customizations/customizations.ts

```diff
@@ -9,7 +10,8 @@ const exampleCardCustomizations: IExampleCardCustomizations[] = [
   { title: 'Word', customizations: WordCustomizations },
   { title: 'Teams', customizations: TeamsCustomizations },
   { title: 'Azure', customizations: AzureCustomizationsLight },
-  { title: 'Azure Dark', customizations: AzureCustomizationsDark }
+  { title: 'Azure Dark', customizations: AzureCustomizationsDark },
+  { title: 'Mdl2', customizations: Mdl2Customizations }
```

Comment:
```suggestion
  { title: 'MDL2', customizations: Mdl2Customizations }
```

Reply:
Even if we don't change the const/type casing, I think it definitely makes sense to have the acronym uppercase in this dropdown.

Reply:
Thanks for suggestions. I was kind of debating on this one. Should I change the file name also?